modules/ROOT/pages/configuration-schema-import-filter.adoc

```diff
@@ -61,25 +61,14 @@ If you are using legacy configuration then, you need to include it in your confi
 | Name of sync gateway database
 
 | scopes 
-| tenant1 
-| Name of scope is tenant1
+| Scope1 
+| Name of scope is Scope1
 
 | Collections: 
```

Comment:
This name is still wrong isn't it?

Reply:
e.g. `Collections:`
We're not explaining what a "Collections" looks like, but an import filter?
(or am I misunderstanding)

Reply:
Yeah I get what you are saying